### ISSUE TITLE
Instruct developers to use a UBI base image.

### ISSUE THREAD

**Issue by hulquest** [operator-framework/operator-sdk#4926]
### What is the URL of the document?

https://sdk.operatorframework.io/docs/building-operators/golang/tutorial

### Which section(s) is the issue in?

https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#configure-the-operators-image-registry

### What needs fixing?

Developers should be instructed to choose a UBI base image if the operator will be made available on OperatorHub.

#### Additional context

I don't think the existing text should be replaced but I suggest adding a line that states the UBI base image should be used if the developer wants to publish the operator at OperatorHub.  Of course, provide a link to the UBI image page in the recommendation as well.

**Comment by estroz** [comment]
@hulquest do you mean getting Red Hat certified requires UBI? I do not think getting onto OperatorHub requires UBI.

/triage support

**Comment by hulquest** [comment]
@estroz You are correct.  If the number of people building certified operators is so small it doesn't warrant a blurb about using a UBI base image, then we can close this.  I was new to the subject and got confused.
